assets/sass/components/settings/_googlesitekit-settings-notice.scss

```
@@ -213,19 +213,19 @@
 }
 
 .googlesitekit-settings-notice-adsense-top-earning-pages-widget {
-	align-items: center;
-	flex-wrap: wrap;
+	align-items: normal;
+	flex-flow: column wrap;
 	padding-bottom: 24px;
 	padding-top: 24px;
 	row-gap: 15px;
 
-	.googlesitekit-settings-notice__icon {
-		margin-right: 20px;
+	@media (min-width: $bp-desktop) {
+		align-items: center;
+		flex-direction: row;
 	}
 
-	.googlesitekit-settings-notice__body {
-		display: flex;
-		flex: 1;
+	.googlesitekit-settings-notice__icon {
+		margin-right: 20px;
```

As per my other comment, introducing a column gap of `15px` means we should drop this down to `5px`:

```suggestion
		margin-right: 5px;
```